ydb/public/sdk/cpp/client/ydb_query/tx.h

```diff
@@ -63,12 +63,16 @@ struct TTxSettings {
         TS_SNAPSHOT_RO
     };
 
-    const ETransactionMode Mode_;
     FLUENT_SETTING(TTxOnlineSettings, OnlineSettings);
 
+    ETransactionMode getMode() const {
```

Comment:
Wrong style, functions should begin with capital letter
```suggestion
    ETransactionMode GetMode() const {
```